tests/python/relay/aot/aot_test_utils.py

```diff
@@ -64,6 +68,38 @@ class AOTTestModel(NamedTuple):
     params: Optional[Dict[str, np.array]] = None
 
 
+class AOTTestRunner(NamedTuple):
+    """Class to describe a test runner for AOT code
+
+    Parameters
+    ----------
+    makefile: str
+        Premade Makefile to use from the AOT test folder
+    prologue: str
+        Code to prepend to the main function
+    includes: List[str]
+        Additional includes required to run the AOT test runner
+    parameters: Map[str, str]
+        Additional parameters to pass to the make command
+    """
+
+    makefile: str = "default"
+    prologue: str = ""
+    includes: List[str] = []
+    parameters: Dict[str, str] = {}
+
+
+AOT_DEFAULT_RUNNER = AOTTestRunner()
+AOT_CORSTONE300_RUNNER = AOTTestRunner(
```

Comment:
Maybe its worth linking here for more info about the reference system ?

Reply:
Great suggestion, I've added this :smile_cat: